data/free worlds reconciliation.txt

```diff
@@ -1732,6 +1809,52 @@ mission "FW Pug 4"
 	on complete
 		event "battle for delta capricorni"
 
+mission "FW Pug 4(Extra Ship)"
```

Comment:
Instead of duplicating this mission, use a `branch` structure to change the necessary bit of the conversation based off of the player having had a JD before.